src/drivers/wdm/queries/OpaqueMdlWrite/driver_snippet.c

```diff
@@ -0,0 +1,83 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+//
+// driver_snippet.c
+//
+
+// Template. Not called in this test.
+void top_level_call() {}
+
+_Use_decl_annotations_
+    NTSTATUS
+    DispatchWrite(
+        PDEVICE_OBJECT DeviceObject,
+        PIRP Irp)
+{
+    UNREFERENCED_PARAMETER(DeviceObject);
+    UNREFERENCED_PARAMETER(Irp);
+    return STATUS_SUCCESS;
+}
+
+_Dispatch_type_(IRP_MJ_SET_INFORMATION)
+    DRIVER_DISPATCH DispatchSetInformation;
+
+_Use_decl_annotations_
+    NTSTATUS
+    DispatchSetInformation(
+        PDEVICE_OBJECT DeviceObject,
+        PIRP Irp)
+{
+    UNREFERENCED_PARAMETER(DeviceObject);
+    UNREFERENCED_PARAMETER(Irp);
+    return STATUS_SUCCESS;
+}
```

Comment:
Remove this section as it's unrelated to the specific test.

Reply:
See general comment on the PR replying to your review comment.